packages/mux-uploader/src/mux-uploader.ts

```diff
@@ -135,6 +134,28 @@ class MuxUploaderElement extends globalThis.HTMLElement implements MuxUploaderEl
     this._endpoint = value;
   }
 
+  get noDrop(): boolean {
+    return this.hasAttribute('noDrop');
```

Comment:
```suggestion
    return this.hasAttribute('nodrop');
```

lets keep the design guidelines to keep attributes lowercase.
it has no effect but still it's unusual